backend.py

```diff
@@ -919,16 +920,17 @@ def get_start_tiles(board, tile_type="start"):
     for coordinate, tiles in board.items():
         for tile in tiles:
             if tile.type == tile_type:
-                robot_tiles[tile.number] = {"coordinates": coordinate,
-                                            "tile_direction": tile.direction}
+                if len(robot_tiles) < int(players):
```

Comment:
Tuhle podmínku moc nechápu. Proč `len(robot_tiles) < int(players)`?

Reply:
To bylo první řešení co mě napadlo,  aby se další startovací políčko vytvořilo pouze v případě, že je jich méně než zadaný počet hráčů.

Reply:
Tož funguje to :)
Jen je tam zbytečné to převádění čísla na číslo.

A taky pro úplnost doporučuju `break`, když zjistíš že nemá smysl pokračovat v cyklu.